docs/pages/component/props.mdx

```diff
@@ -97,7 +115,7 @@ Configure buffering / data loading strategy.
 To provide a custom chapter source for tvOS. This prop takes an array of objects with the properties listed below.
 
 | Property  | Type    | Description                                                                                                                                               |
-| --------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|-----------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
same here : can you remove these patch please ?
No need that you become owner of these line !